speedtest/unit.go

```diff
@@ -45,17 +46,25 @@ type ByteRate float64
 
 var globalByteRateUnit UnitType
 
+func (r ByteRate) MarshalJSON() ([]byte, error) {
+	if rStr := r.String(); rStr != naString {
+		return []byte(strings.Split(rStr, " ")[0]), nil
```

Comment:
Since the unit is auto-scaled, the user cannot know what the unit is.

Reply:
I see, I tried including the unit but it errors out, probably because it expects a float instead of a string, that's why I did the split to get rid of the unit. I could add a new field to the server struct for DLString and ULString, and make the original ones not appear in the JSON. I'll get an example to get your feedback.